torchvision/datasets/fgvc_aircraft.py

```diff
@@ -12,15 +12,15 @@
 class FGVCAircraft(VisionDataset):
     """`FGVC Aircraft <https://www.robots.ox.ac.uk/~vgg/data/fgvc-aircraft/>`_ Dataset.
 
-    The dataset contains 10,200 images of aircraft, with 100 images for each of 102
+    The dataset contains 10,000 images of aircraft, with 100 images for each of 100
```

Comment:
Oh boy, they even got the number of samples wrong on their website? :roll_eyes: Thanks for checking and fixing!

Reply:
If the website is reporting the original values, are we doing something wrong in the code?

Reply:
I don't think so, no:

```shell
$ cat variants.txt | wc -l
100
$ cat families.txt | wc -l
70
$ cat manufacturers.txt | wc -l
30
$ ls images | wc -l
10000